src/main/kotlin/uk/gov/gdx/datashare/services/DataReceiverService.kt

```diff
@@ -59,7 +59,7 @@ class DataReceiverService(
     log.debug(
       "Notifying Data Processor of event type {} from {}",
       dataProcessorMessage.eventType,
-      publisher.name,
+      supplier.name,
     )
 
     meterRegistry.counter("EVENT_ACTION.EventPublished", "eventType", eventPayload.eventType.name).increment()
```

Comment:
I assume we want to keep this over e.g. EventSupplied?

Reply:
I thought so, but we can change later if we want